docs/source/index.rst

```diff
@@ -6,12 +6,87 @@
-enhanced-sampling molecular dynamics simulations to discover reaction pathways and estimate
-free energies.
+PySAGES addresses these challenges by offering a python interface between highly optimized simulation engines and the researcher to implement collective variables and enhanced-sampling methods.
+Even better, PySAGES already provides an extensible framework to compute collective variables and to perform enhanced-sampling MD simulations to discover reaction pathways and estimate free energies.
+Most research objectives are achievable by using these implemented collective variables and methods.
+PySAGES currently supports automatically connecting these methods to `HOOMD-blue <https://glotzerlab.engin.umich.edu/hoomd-blue>`__ and
+`OpenMM <http://openmm.org/>`__.
+Both engines offer a python interface to the user and implement the simulation on GPUs for best performance.
+PySAGES interacts with both backends directly on the GPU memory; copying between GPU and host memory is not required.
+This approach allows biased simulations without slowing the backend simulation engines down.
+PySAGES still implements all methods and collective variables as pure python for access and modification.
+
+
+Getting Started
+---------------
+
+PySAGES is designed to seamlessly integrate with your existing simulations.
+For an approachable start we prepared examples in Google Colab documents, which provide easy access to a working simulation environment and GPU hardware.
+For an example of using PySAGES with HOOMD-blue, check out `this <https://colab.research.google.com/github/SSAGESLabs/PySAGES/blob/main/examples/hoomd-blue/Umbrella_Integration.ipynb>`__ example for simple umbrella integration.
+If you are working with OpenMM, we offer `this <https://colab.research.google.com/github/SSAGESLabs/PySAGES/blob/main/examples/openmm/Harmonic_Bias.ipynb>`__ notebook, which explores the free energy landscape of the conformational space of alanine dipeptide via harmonic biasing.
+Using the notebooks enables you to get started with PySAGES without installation.
+For a productive deployment, we recommend a full installation.
+
+We offer more examples, in the form of both Google Colab notebooks and for fully installed PySAGES, in the GitHub repository `examples <https://github.com/SSAGESLabs/PySAGES/tree/main/examples>`__.
+
+
+Installation
+------------
+
+The dependencies for a PySAGES installation depend on your desired simulation backend engine.
+Depending on your system setup, it may be necessary to reinstall the simulation backend, especially if you do not have write permission for your current installation.
+On top of the current installation, it is required that you install a plugin that connects PySAGES with the simulation engine:
+
+ - For HOOMD-blue, visit `HOOMD DLPack Plugin <https://github.com/SSAGESLabs/hoomd-dlext>`__.
+ - For OpenMM, go to `OpenMM DLPack Plugin <https://github.com/SSAGESLabs/openmm-dlext>`__.
+
+And follow their installation instructions.
```

Comment:
```suggestion
and follow their installation instructions.
```

Reply:
We should remove the periods after the hyperlinks if we do that.